pipeline/terraform/main.tf

```diff
@@ -34,18 +34,26 @@ module "catalogue_pipeline_20200228" {
     local.mets_adapter_topic_arn,
   ]
 
+  calm_adapter_topic_arns = [
+    //    local.mets_reindexer_topic_arn,
```

Comment:
Should this be a calm_reindexer_topic_arn?

Reply:
Yarp.